modules/btcpay/src/Installer/Hooks.php

```diff
@@ -21,7 +21,6 @@ public function install(): array
 		if (!$this->module->registerHook('displayAdminOrderMainBottom')
 			|| !$this->module->registerHook('displayOrderDetail')
 			|| !$this->module->registerHook('displayPaymentEU')
-			|| !$this->module->registerHook('payment')
```

Comment:
Good catch, we no longer have that hook. That does mean we need to add a migration to remove it when people upgrade to this version. I'll take care of that over the weekend.

Reply:
I can also take a look on how that migration would work if you want.